letsencrypt-auto-source/pieces/bootstrappers/rpm_common.sh

```diff
@@ -25,9 +25,9 @@ BootstrapRpmCommon() {
   fi
 
   if ! $SUDO $tool list *virtualenv >/dev/null 2>&1; then
-    echo "To use Certbot, packages from the EPEL repository need to be installed."
+    say "To use Certbot, packages from the EPEL repository need to be installed."
```

Comment:
Well, we might be able to do `sudo -A ...` and use the `-A` argument to spit out stuff we should have printed earlier... I'm not sure how portable `-A` is...